### ISSUE TITLE
Lock focus on Datastore build screen in GUI

### ISSUE THREAD

**Issue by kmdouglass** [kmdouglass/bstore#54]
The focus on the Datastore build screen should not change. Currently it does, which allows users to open multiple build screens and is not desirable.

Also to do with the GUI:
+ Finalize the layout of the filename and search path frames.
+ Lock the focus on the parser configure screen when it's open

These changes should be added to the gui-dev branch before merging into development.

**Comment by kmdouglass** [comment]
Everything is fixed except for keeping focus on the parser configuration modal dialog. I honestly can't find an easy way to create nested modal dialogs with Tk, so I will just trust the user on this one to not continuously click the parser radio button.

I will merge into development once a bit more tidying of the code is complete.
